appautoscaling.tf

```diff
@@ -26,3 +26,23 @@ resource "aws_appautoscaling_policy" "scale_cpu" {
     }
   }
 }
+
+resource "aws_appautoscaling_policy" "scale_mem" {
```

Comment:
This resource name can be changed to `scale_memory`.

Reply:
Adjusted